src/bap_mc/bap_mc.ml

```diff
@@ -170,6 +201,22 @@ module Cmdline = struct
     Arg.(value & opt_all ~vopt:`bil (enum formats) [] &
          info ["show-bil"] ~doc)
 
+  let addr_offset =
+    let doc = sprintf
+        "Specify a pretend offset, as though the instructions\
+	 occur at a certain address, and accordingly \
+	 interpreted. Bases can be hex or decimal " in
+    Arg.(value & opt ~vopt:0 (int) 0 &
+	 info ["offset"] ~doc)    
+
+  let stop_at =
+    let doc = sprintf
+		"Specify a number of instructions to disassemble.\
+		 Good for ensuring that only one instruction is ever\
+		 lifted or disassembled from a byte blob. 0 connotes all" in
+    Arg.(value & opt ~vopt:0 (int) 0 &
```

Comment:
Instead of using `0` as a sentinel for infinity it is better to use `None` to specify the absence of the parameter, and use it as a flag that we should dissassemble as many as possible.